Dockerfile.freyja_apps

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+# SPDX-License-Identifier: MIT
+
+# syntax=docker/dockerfile:1
+
+# Comments are provided throughout this file to help you get started.
+# If you need more help, visit the Dockerfile reference guide at
+# https://docs.docker.com/engine/reference/builder/
+
+################################################################################
+# Create a stage for building the application.
+
+ARG RUST_VERSION=1.72.1
```

Comment:
can you update the toolchain file to match? this repo is currently on stable

Reply:
Okay so I dont think it is as simple as putting 'stable' in there, but I believe if I change the docker image to docker.io/library/rust:slim-bullseye it will be equivalent to stable based on this doc: https://hub.docker.com/_/rust?tab=tags
Let me know if you would still like me to make the change

Reply:
sorry, I mean update the toolchain file to point to 1.72.1

Reply:
updated toolchain file